kodilla-good-patterns/src/main/java/ThirdChallenge/FlightFinder.java

```diff
@@ -0,0 +1,170 @@
+package ThirdChallenge;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.stream.*;
+
+public class FlightFinder implements Communications {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
nie rób z tego statycznej :-) brzydko wygląda haha
private final Scanner console = new Scanner(System.in);